src/Services/LightningService.cs

```diff
@@ -1312,15 +1312,6 @@ await foreach (var response in closeChannelResult.ResponseStream.ReadAllAsync())
                     var localBalance = channel.LocalBalance + htlcsLocal;
                     var remoteBalance = channel.RemoteBalance + htlcsRemote;
 
-                    // If the channel is not initiated by a NodeGuard node, we need to swap the balances.
-                    // the balance is always shown from the NodeGuard's perspective
-                    if (!channel.Initiator)
-                    {
-                        localBalance = channel.RemoteBalance + htlcsRemote;
-                        remoteBalance = channel.LocalBalance + htlcsLocal;
-                    }
-
-
```

Comment:
we're always looking through the POV of the managed node, so local balance doesn't have to be reversed